### PR TITLE
[PHP 8.4] intl update part 2

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

**Comment by devnexen** [comment]
for #3872

File: reference/intl/intlcalendar/set.xml [hunk 1]
```diff
@@ -156,6 +156,13 @@
      </thead>
      <tbody>
       &return.type.true;
+      <row>
+       <entry>8.4.0</entry>
+       <entry>
+        This had been deprecated in favor of the methods
+        <methodname>setDate</methodname> and <methodname>setDateTime</methodname>.
```
Comment (Girgias) [inline reference/intl/intlcalendar/set.xml:163]:
```suggestion
        <methodname>IntlCalendar::setDate</methodname> and
        <methodname>IntlCalendar::setDateTime</methodname>.
```

File: reference/intl/intlgregoriancalendar/construct.xml [hunk 2]
```diff
@@ -107,6 +107,31 @@
...
+       <entry>&Version;</entry>
+       <entry>&Description;</entry>
+      </row>
+     </thead>
+     <tbody>
+      <row>
+       <entry>8.4.0</entry>
+       <entry>
+        This had been deprecated in favor of the methods
+        <methodname>createFromDate</methodname> and <methodname>createFromDateTime</methodname>.
```
Comment (Girgias) [inline reference/intl/intlgregoriancalendar/construct.xml:126]:
```suggestion
        This had been deprecated in favor of the methods
        <methodname>IntlGregorianCalendar::createFromDate</methodname> and
        <methodname>IntlGregorianCalendar::createFromDateTime</methodname>.
```

File: reference/intl/intlgregoriancalendar/construct.xml [hunk 3]
```diff
@@ -107,6 +107,31 @@
   </variablelist>
  </refsect1>
 
+ <refsect1 role="changelog">
+  &reftitle.changelog;
+  <para>
+   <informaltable>
```
Comment (Girgias) [inline reference/intl/intlgregoriancalendar/construct.xml:113]: Remove wrapping para tag

File: reference/intl/numberformatter-constants.xml [hunk 4]
```diff
@@ -641,6 +641,16 @@
    and <function>numfmt_set_attribute</function> with
    <constant>NumberFormatter::ROUNDING_MODE</constant> attribute.
    <variablelist>
+
+    <varlistentry xml:id="numberformatter.constants.round-away-from-zero">
+     <term>
+      <constant>NumberFormatter::ROUND_AWAY_FROM_ZERO</constant>
+     </term>
+     <listitem>
+      <simpara><constant>ROUND_UP</constant> alias.</simpara>
```
Comment (Girgias) [inline reference/intl/numberformatter-constants.xml:650]:
```suggestion
      <simpara>
       &Alias; <constant>NumberFormatter::ROUND_UP</constant>.
      </simpara>
```

File: reference/intl/numberformatter-constants.xml [hunk 5]
```diff
@@ -705,6 +726,15 @@
      </listitem>
     </varlistentry>
 
+    <varlistentry xml:id="numberformatter.constants.round-towards-zero">
+     <term>
+      <constant>NumberFormatter::ROUND_TOWARDS_ZERO</constant>
+     </term>
+     <listitem>
+      <simpara><constant>ROUND_DOWN</constant> alias.</simpara>
```
Comment (Girgias) [inline reference/intl/numberformatter-constants.xml:734]: Ditto

**Review by Girgias — APPROVED** [review]

File: reference/intl/numberformatter-constants.xml [hunk 6]
```diff
@@ -705,6 +726,15 @@
      </listitem>
     </varlistentry>
 
+    <varlistentry xml:id="numberformatter.constants.round-towards-zero">
+     <term>
+      <constant>NumberFormatter::ROUND_TOWARDS_ZERO</constant>
```
Comment (kocsismate) [inline reference/intl/numberformatter-constants.xml:731]: There is no `ROUND_TOWARDS_ZERO` constant: it is `ROUND_TOWARD_ZERO` in fact.

**Pull request merged**